presto-spi/src/main/java/com/facebook/presto/spi/function/WindowIndex.java

```diff
@@ -72,6 +73,22 @@
     Slice getSlice(int channel, int position);
 
     /**
+     * Gets a value stored as a {@link Block}.
+     * @param channel argument number
```

Comment:
Newline between description and first `@param`. Fill in `@return` description or remove it.